src/Models/ShallowWaterModels/shallow_water_model.jl

```diff
@@ -74,6 +74,8 @@ function ShallowWaterModel(;
 
     tracers = tupleit(tracers) # supports tracers=:c keyword argument (for example)
 
+    @assert topology(grid)[3] == Flat "ShallowWaterModel must be Flat in the vertical.  Please fix and rerun."
```

Comment:
```suggestion
    @assert topology(grid, 3) === Flat "ShallowWaterModel requires `topology(grid, 3) === Flat`. Use `topology = ($(topology(grid, 1)), $(topology(grid, 2)), Flat)` when constructing `grid`."
```

Reply:
Tests no longer pass.  I wonder if there was a typo above?

Reply:
I tested the new line and I believe it is correct, so will return to this being resolved.